src/site/topic26.rst

```diff
@@ -162,7 +162,7 @@ Quicksort
 
 
 * There are two important ideas at the root of quicksort
-    1. Given a list of sorted values :math:`< x`, and another list of sorted values :math:`> x`, one can append the second list to the first to create a larger sorted list
+    1. Given a list of sorted values :math:`< x`, and another list of sorted values :math:`> x`, one can concatenate the first and second lists to create a larger sorted list
```

Comment:
re: append vs concatenate. Surprisingly, I've put thought into this in the past. Writing libraries makes you think about weird things. Here was my conclusion.

Append is an _operation_; a _mutation_, whereas concat is a _function_.  
That is, append should always be something like:
```java
class ListMe<T> {
    boolean append(T... items) { /*... */ }
}
```
And concat should be:
```java
final class Lists {
    static List<T> concat<T>(List<T> first, List<T> second) { /* ... */ }
}
```

All that's to say. I approve of the wording change.